src/plumber.R

```diff
@@ -80,7 +96,8 @@ function(ensemblId, efoId) {
 #* @get /tpm/gene-disease-gtex/plot
```

Comment:
Since producing plots can take a while, i'd consider attempting to parallelize the plot endpoints so that plot requests don't eat up all the processes. See the plumber [1.1.0 release notes](https://blog.rstudio.com/2021/03/29/plumber-v1-1-0/#parallel-exec) and the [article](https://rstudio.github.io/promises/articles/future_promise.html) those notes point to. 

Granted, the plot endpoint _will_ take longer to send responses to users because there's more to send in plots, but still somthing to consider.

Reply:
Could you submit an issue for this suggestion with more implementation details, or submit a PR for implementing this?

I think this feature would be more properly implemented as a standalone PR, because it is not trivial, and we are planning to merge this PR in the next few days. I think implementing this feature would take at least the following steps:

- Read the documentations of `promises` package and `plumbr` parallel execution model completely. I was confused by `plumbr` documentation claiming that requests are handled sequentially at <https://www.rplumber.io/articles/execution-model.html#performance-request-processing>. Could you specify the `plumbr` parallel execution model in the upcoming issue/PR?
- Check source code if documentations are unclear.
- Implement this feature.
- Test this feature. Check whether requests are actually executed concurrently.
- Discuss with DevOps team how this would affect previously designed Amazon Fargate/ECS/etc scaling procedure.

Reply:
Fair enough - yeah this may be too big for this pr. I'm wondering if @blackdenc may be able to speak to any load balancing or anything on the infra side to handle concurrent requests ...